### DISCUSSION TITLE
How to use important modifier in SCSS with `@apply`?

### DISCUSSION THREAD

**Discussion by brc-dd** [tailwindlabs/tailwindcss#4633] [Help]
I want to do something like this:

```scss
// https://tailwindcss.com/docs/just-in-time-mode#built-in-important-modifier
.foo {
    @apply !w-[25rem] !h-auto;
}
```

But it is not working, and throwing error. As a workaround I am able to use interpolation:

```scss
// https://tailwindcss.com/docs/using-with-preprocessors#sass
.foo {
    @apply w-[25rem] h-auto #{!important};
}
```

Although the latter one works, but in it, I need to write the non-important classes on a different line. It would be great if I can specify them on the same line, without using `@apply` again.

**Comment by qortex** [comment]
Thanks, works for me as my use case is less demanding than yours.
VSCode SCSS parser outputs a syntax error though: `at-rule or selector expectedscss(css-ruleorselectorexpected)`
